pkg/cli/clients/directory_client.go

```diff
@@ -16,9 +16,10 @@ import (
 type Config struct {
 	Host      string `flag:"host" short:"H" help:"directory service address" env:"TOPAZ_DIRECTORY_SVC" default:"localhost:9292"`
 	APIKey    string `flag:"api-key" short:"k" help:"directory API key" env:"TOPAZ_DIRECTORY_KEY"`
-	Insecure  bool   `flag:"insecure" short:"i" help:"skip TLS verification"`
-	SessionID string `flag:"session-id"  help:""`
-	TenantID  string `flag:"tenant-id" help:""`
+	Token     string `flag:"token" short:"t" help:"JWT used for connection" env:"TOPAZ_DIRECTORY_TOKEN"`
+	Insecure  bool   `flag:"insecure" short:"i" help:"skip TLS verification" env:"TOPAZ_DIRECTORY_INSECURE"`
+	SessionID string `flag:"session-id"  help:"" env:"TOPAZ_SESSION_ID"`
```

Comment:
I don't think we need SessionID anymore. We now use etags to do conditional updates.
And if we remove SessionID, the authorizer and directory config structs become identical. The only differences are the env vars names. Perhaps we can combine the two structs and use kong's `envprefix` option to differentiate the env vars.